initializr-generator-spring/src/main/resources/templates/documentation/docker-compose.mustache

```diff
@@ -0,0 +1,9 @@
+### Docker Compose support
+
+This project contains a Docker Compose file.
```

Comment:
I can see the customizer does not generate any section if the services are empty. I don't think that's right. We want adding docker-compose to be the trigger of the doc. If no matching service has been found in what the user has selected, this is an important piece of information that should be shared, rather than not writing the section at all.

Reply:
BTW, I got things mixed up between lib and start.spring.io. 

I think we should trigger the contributor only if we want to use it and it should therefore behave as I've explained. If you don't want the contributor, guard it with a condition. This way, if it is around, it should warn you no service has been created IMO.

Reply:
I've now created a `DockerComposeProjectGenerationConfiguration` which lives in the intializr and can be `@Import`ed from the site if the `docker-compose` is requested.

If no services have been defined, a warning is added and the docker compose section changes.